tools/project_manager/add_files.php

```diff
@@ -824,12 +824,12 @@ function display()
                     echo "</td>";
 
                     // action
-                    $bgcolors = array(
-                        ''        => '#ffffff',
-                        'add'     => '#ccffcc',
-                        'replace' => '#ffccaa',
-                        'same'    => '#ffffff',
-                        'error'   => '#ffcccc',
+                    $bgclass = array(
```

Comment:
Right now this class just changes the background of the element, but it isn't restricted to only that. Consider renaming this just `$class` or `$addl_class` or somesuch to separate it from the "background" aspect.

Reply:
Done.